src/main/resources/application.properties

```diff
@@ -56,6 +56,7 @@ spring.rabbitmq.host=localhost
 spring.rabbitmq.port=5672
 spring.rabbitmq.username=guest
 spring.rabbitmq.password=guest
+management.health.rabbit.enabled=false
```

Comment:
Disable the health checker since it's no need to check RabbitMQ health when ActiveMQ or Kafka is used in PDA. And even RabbitMQ is used in PDA it is also uncessary since there is error message when register listners when RabbitMQ  server is unavailable.